meerkat_abacus/config.py

```diff
@@ -62,6 +62,11 @@ def __init__(self):
         self.db_dump_folder = '/var/www/dumps/'
 
         self.import_fraction = float(os.environ.get("IMPORT_FRACTION", 0))
+
+        self.only_import_after = os.environ.get("ONLY_IMPORT_AFTER", None)
+        if self.only_import_after:
+            self.only_import_after = parse(self.only_import_after)
+        print(self.only_import_after)
```

Comment:
Could you create a temporary variable for the env variable and skip mutating the `self.only_import_after` field?
Also change the print to logging statement.

Reply:
Why is mutating the variable a problem?
Do you want

```
self.only_import_after = None
tmp = os.environ.get("ONLY_IMPORT_AFTER", None)
if tmp:
          self.only_import_after = parse(tmp)
```

or 

```

tmp = os.environ.get("ONLY_IMPORT_AFTER", None)
if tmp:
         self.only_import_after = parse(tmp)
else: 
        self.only_import_after = None
```

Reply:
The 2nd looks better.
Maybe I've used wrong words. Mutating may be fine but it's better to avoid it if possible. It's less prone to create bugs especially if not only you mutate the value but also the type of a variable.
Also I'd suggest renaming the field to `self.only_import_after_date`.

Reply:
I think I disagree about this, but will change it.